ConsoleApplication/Helpers/GraphClientFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DeltaQueryApplication
+{
+    public static class GraphClientFactory
```

Comment:
You probably want to add XML comments on the class and public methods to explain what this does?